### DISCUSSION TITLE
brypt! Comment in PR

### DISCUSSION THREAD

**Discussion by kaaax0815** [coder/code-server#3572] [Q&A] [answered]
I saw [this](https://github.com/cdr/code-server/pull/3422#pullrequestreview-664873712) comment

> bcrypt!

I dont understand it because argon2 is better then bcrypt isnt it? why should it be used then? or am i completely misunderstanding this? I know its a weird question but somewhy it bothers me lul

**Comment by jsjoeio** [comment, marked as answer]
Lol definitely lacking context! Originally, I shared the problem and our first solution with Ammar. He said he liked `bcrypt` and was in favor.

Then we chatted with more folks and heard suggestions to use `argon2` which as you say and they said, is better. 

We ended up going with that approach so that comment is now outdated lol I hope that helps!

**Reply by kaaax0815** [reply]
Thanks for the context xd
